### ISSUE TITLE
Accessibility Report: Brand Bar Component

### ISSUE THREAD

**Issue by sherakama** [SU-SWS/decanter#314]
Use the AMP tool to scan the [Brand Bar](http://decanter.stanford.edu/section-components.html#kssref-components-brand-bar) component for accessibility issues and document them to this ticket.

**Comment by josephgknox** [comment]
![Screen Shot 2019-03-15 at 12 51 57 PM](https://user-images.githubusercontent.com/8933670/54458427-2d3d5d80-4721-11e9-8a0e-a13de6a7c6ec.png)
![Screen Shot 2019-03-15 at 12 52 07 PM](https://user-images.githubusercontent.com/8933670/54458428-2f072100-4721-11e9-9641-ae7769a77ec9.png)

**Comment by josephgknox** [comment]
There are five violations, all of which are the same.

**Comment by yvonnetangsu** [comment]
Great, that means the PR to remove brand bar ID yesterday should have taken care of this issue. I guess we'll push out a new build of the website before 5.0 so we're fine :)
